Opdate references to dependency injection (#6328)
umbraco/UmbracoDocs: 14/umbraco-cms/reference/security/serverside-sanitizing.md

# Sanitizing the Rich Text Editor

The Rich Text Editor is sanitized on the frontend by default, however, you may want to do this serverside as well. The libraries that are out there tend to have strict dependencies. That is why we will leave it up to you how you want to sanitize the HTML.

## Implementing your own IHtmlSanitizer

> We have added an abstraction called `IHtmlSanitizer` which by default this does not do anything. You can overwrite it with your own implementation to handle sanitization how you see fit. This interface has a single method: `string Sanitize(string html)`. The output of this method is what will be stored in the database when you save a Rich Text Editor.

Reviewer: ```suggestion
We have added an abstraction called `IHtmlSanitizer`, which by default does nothing. You can override it with your own implementation to handle sanitization as you see fit. This interface has a single method: `string Sanitize(string html)`. The output of this method is what will be stored in the database when you save a Rich Text Editor.
```
